### DISCUSSION TITLE
woowacourse/woowacourse-level5-github-oauth-transfer/

### DISCUSSION THREAD

**Discussion by giscus** [chlwlstlf/chlwlstlf.github.io#5] [Announcements]
# woowacourse/woowacourse-level5-github-oauth-transfer/

코레아 계정으로 Github OAuth transfer 하기

https://chlwlstlf.github.io/woowacourse/woowacourse-level5-github-oauth-transfer/

**Comment by ooherin** [comment]
핫한 블로그라고 해서 놀러왔습니다! 글 열심히 쓰시네요:) 가끔 올게요 ㅎㅎ

**Reply by chlwlstlf** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ누구 피셜이에욬ㅋㅋㅋㅋㅋㅋㅋ 아휴 감사합니다 저도 놀러갈게요

**Comment by chysis** [comment]
모든 유저 정보가 제대로 넘어갔네요! GitHub docs에 관련 내용을 좀 더 자세히 적어줬으면 하는 바람이 있네요

**Reply by chlwlstlf** [reply]
깃허브 docs 평소엔 되게 친절한데 가끔 이렇게 불친절하면 서운합니다.
